app/utils.py

```diff
@@ -1168,6 +1168,21 @@ def __init__(self, request, response, env):
                 self.render('not_admin_error.html', logout_url=logout_url, user=user)
                 self.terminate_response()
                 return
+            self.env.logout_url = users.create_logout_url(self.request.url)
+            try:
+                # This is different from env.repo_options because this contains
+                # all repositories including deactivated ones.
+                self.env.all_repo_options = [
```

Comment:
Why would this throw an exception? It's not clear what's dangerous here.

Reply:
Hm, yeah I don't know; the original code did this but it's not clear why. Also, now that I think about it, the reason to log the exception like this doesn't apply anymore because this is now evaluated at the same time as the rest of the code here runs (rather than from within the template). I've removed the try-except.